# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- thunderbird-128.0.1/toolkit/mozapps/update/updater/updater.cpp.vanilla	2024-08-01 22:15:05.881448120 +0200
+++ thunderbird-128.0.1/toolkit/mozapps/update/updater/updater.cpp	2024-08-01 22:15:34.312448209 +0200
@@ -107,9 +107,9 @@
 #  define USE_EXECV
 #endif
 
-#if defined(XP_OPENBSD)
+//#if defined(XP_OPENBSD)
 #  define stat64 stat
-#endif
+//#endif
 
 #if defined(MOZ_VERIFY_MAR_SIGNATURE) && defined(MAR_NSS)
 #  include "nss.h"
